RELEASING.md

```diff
@@ -280,7 +272,13 @@ See the documentation on the [JanusGraph release policy](https://docs.janusgraph
 
 *This is the point of no return.* After releasing artifacts to Maven
 Central and pushing history to the public GitHub repo, the release
-can't be canceled.
+can't be canceled. 
+
+Update the `STRUCTOR_LATEST_TAG` variable on Travis-CI and 
```

Comment:
(nitpick) Not sure if _on Travis-CI_ is clear enough so everyone knows that the web interface is meant and not `.travis.yml`. Maybe replace this with _in the settings of the JanusGraph project in the Travis CI web interface_.